Abstracts/Events/Interfaces/ShouldHandleNow.php

```diff
@@ -7,7 +7,7 @@
  *
  * @author  Arthur Devious
  */
-interface ShouldHandleNow extends ShouldHandle
+interface ShouldHandleNow
```

Comment:
What is your opinion on this:
```
interface ShouldHandleEvents
interface ShouldHandleEventsImmediately extends ShouldHandleEvents
interface ShouldHandleEventsQueued extends ShouldHandleEvents
```

Would that be a `Breaking Change`?

Reply:
well people who have implemented the old interfaces will have to adjust their code. So when they update apiato to the latest version their code won't work anymore until they've migrated to the new interfaces

Reply:
that is true, yes.. hm - you're right ;)

Reply:
we can deprecate if you want but i think the naming is fine :p can you merge the pull request?

Reply:
johannes would you be so kind to merge this pull request to the master :p and release a new version ?